Future Plan.md

```diff
@@ -0,0 +1,61 @@
+
+1. Achieve full standards compliance - we recognise that Chakracore is significantly behind in some areas due to its slowed pace of development, it may take some time to catch up but we will aim to do it. Any deviation from standards should be considered a bug and we will aim to fix it.
+
+1. Improve performance - we will almost certainly not be able to keep up with v8 or JSC on performance due to the smaller size of our team and limited time we have to work on this, but we will aim to continue to improve performance when we can.
+
+1. Invite community involvement - this project is being continued by volunteers and input will be appreciated from any other volunteers with relevant skills and availability. We will aim to make the project easy to get involved in - if as a potential contributor you see any obvious barriers to entry please raise an issue to discuss them.
+
+1. Simplify the codebase and test suite - when time permits and where practicable we will endeavour to perform housekeeping actions such as removing dead code, combining related tests and removing unnecessary layering.
+
+## Core Community Team
+
+The core team currently planning to work on this are:
+
+- Richard Lawrence (@rhuanjl) - team coordinator (by virtue of having the most free time)
+- Kevin Smith (@zenparsing)
+- Sergey Rubanov (@chicoxyzzy) - JS standards compliance
+- Petr Penzin (@ppenzin) - WASM
+- Alex Syrotenko (@Fly-Style)
+
+## Development process
+
+This community development will proceed on the Chakracore master branch. At a future time this repository may be transferred out of the microsoft organisation within Github - though it will remain here to begin with.
+
+We will periodically (timescale TBD) publish a release plan document listing planned enhancements and proposed timescale for the next release.
+
+We will also aim to triage and address bug reports as received, depending on the scale of these issues they may be fixed immediately or left until a future release.
+
+Pull requests will be invited from anyone who wishes to contribute in addition to the Core Community Team. Any pull request will be required to pass all CI checks and be reviewed by a member of the Core Community Team (different to its author) prior to being merged. The reviewer will merge the PR when happy with it - no PR should be merged by its author.
+
+Additionally any bug fix PR must include a test case that it has fixed; and any new feature PR must include tests for that feature.
+
+Further documentation and guidance will be created on PR best practises including how to author tests - in its absence please ask if unsure.
+
+## Continuous Integration (CI)
+
+We will continue to use the existing check in CI for the time being. However we will aim to migrate this to a platform managed by the community team before March 2021 - we will seek Microsoft's assistance with this migration.
+
+We will additionally ask Google's project OSS-Fuzz to provide continuous fuzzing for our work.
```

Comment:
@divmain I assume this is acceptable for now? Please can you also confirm what level of Azure pipelines support the existing Chakracore CI uses - does it definitely fit within the free tier?

Reply:
This is absolutely acceptable - please incorporate whatever tools make your job easier.  I don't have direct knowledge of ChakraCore CI usage, and whether it fits into the free tier.  That's something I can check on, as well.